docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -96,14 +96,55 @@ releases of Ansible, there can sometimes be exceptions for critical issues.
 
 .. _GitHub: https://github.com/ansible/ansible
 
+.. _changelogs_how_to:
+
 Changelogs
 ~~~~~~~~~~~~~~~~~~
 
-Since 2.5, we've logged changes to ``stable-<version>`` git branches at ``stable-<version>/changelogs/CHANGELOG-v<version>.rst``.
-For example, here's the changelog for 2.5_ on GitHub.
+Since 2.5, we have switched to ``reno`` for generating changelogs based on fragments. When creating new features or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the commit message.
```

Comment:
We're using our own changelog processor in 2.6+ instead of reno. It will most likely be backported to 2.5 as well. It does use reno formatted changelog fragments though.

Details for new plugins and modules are extracted from ansible-doc, not from commit messages.